server/db/dbConfig.js

```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const connectionString = {};
+
+connectionString.development = {
+  user: process.env.USER,
+  host: process.env.HOST,
+  database: process.env.DATABASE,
+  password: process.env.PASSWORD,
+  port: process.env.DB_PORT,
+};
+
+
+connectionString.production = {
+  user: process.env.USER,
+  host: process.env.HOST,
+  database: process.env.DATABASE,
+  password: process.env.PASSWORD,
+  port: process.env.DB_PORT,
+};
+
```

Comment:
Great job, but you should also cater for the test environment

Reply:
okay